app/services/encrypted_attribute.rb

```diff
@@ -41,9 +41,7 @@ def decrypt(cost)
     encryptor = Pii::PasswordEncryptor.new
     decrypted = try_decrypt_with_uak(encryptor) if user_access_key.present?
     return decrypted if decrypted
-    decrypted = try_decrypt_with_all_keys(encryptor, cost)
-    return decrypted if decrypted
```

Comment:
I removed this line. Did you mean line 43? Or did you mean instead of reducing these 2 lines to one, keep them as 2 lines and change line 45? The one-liner does the same thing, more simply :)